packages/flutter/lib/src/material/button_style_button.dart

```diff
@@ -502,7 +502,7 @@ class _ButtonStyleState extends State<ButtonStyleButton> with TickerProviderStat
       onLongPress: widget.onLongPress,
       onHover: widget.onHover,
       mouseCursor: mouseCursor,
-      enableFeedback: resolvedEnableFeedback,
+      enableFeedback: resolvedEnableFeedback ?? true,
```

Comment:
nit: I think it'd be better to add `?? true` to the declaration site for `resolvedEnableFeedback`.